app/domain/hearing-requirements.ts

```diff
@@ -0,0 +1,3 @@
+interface HearingRequirements {
```

Comment:
Can we move this to types folder to have all the interfaces and types in one place?